pages/getting-started/first-steps-with-docker.mdx

```diff
@@ -467,125 +203,79 @@ CONTAINER ID   IMAGE                    COMMAND                  CREATED
-  searches for the specified image in the local repository. If the image is not
-  found locally, Docker fetches it from Docker Hub. Once the image is obtained,
-  Docker creates a new container from this image and initiates it. This command
-  is used for setting up a new container instance with various configurations.
+```
+docker cp bb3de2634afe:/etc/memgraph/memgraph.conf memgraph.conf
+```
 
-- **`docker start`**: This command is used for restarting an
-  existing, previously stopped container. Containers may be stopped using the
-  `docker stop` command or may exit naturally. When a container is stopped, its
-  state is preserved, allowing `docker start` to reactivate the container with
-  all its prior settings intact. This command is used for reactivating a
-  container without repeating the initial setup and initialization process
-  required by `docker run`. It's important to note that `docker start` is
-  applicable only to containers that have already been created and stopped; it
-  cannot be used to create a new container.
+This will save `memgraph.conf` in the current directory.
 
-## Containers and data persistence
+### Copy files to a Docker container
 
-When you're working with Docker, it's essential to understand how data is
-handled in containers. 
+To copy a file from your local system into a container:
+1. Know the container’s ID or name.
+2. Be in the local directory where the file is located.
+3. Use the command:
+```
+docker cp <local_file> <CONTAINER_ID>:<path_in_container>
+```
 
-**Your data is safe**: Even if you [stop a container](#stop-container) using the `docker stop`
-command or if it crashes unexpectedly, the data and changes inside that
-container are retained. So, when you restart the stopped container, you'll find
-everything just as you left it.
+**Example**: Replacing the configuration file inside the container:
 
-However, there are a few things to keep in mind:
+```
+docker cp memgraph.conf bb3de2634afe:/etc/memgraph/memgraph.conf
+```
 
-- **Deleting a container**: If you deliberately delete a container using
-  commands like `docker rm`, the changes and data within that container will be
-  lost unless you've saved them in Docker volumes or bind mounts.
-  
-- **Using volumes**: To ensure long-term data safety and for scenarios where you
-  might delete containers but want to retain their data, Docker provides a
-  feature called
-  [volumes](/getting-started/install-memgraph/docker#specify-volumes). Volumes
-  allow you to store data outside the container, ensuring it remains safe and
-  accessible even if the container is removed.
-
-In short, while working with containers, always remember: stopping isn't
-deleting. Your data stays intact when a container is merely stopped. If you're
-planning to remove containers but wish to keep the data, ensure you've set up
-Docker volumes **prior to running the container**.
+This will overwrite the file inside the container with your local version.
```

Comment:
Maybe it would be useful to mention when this is commonly used with Memgraph - if you want Memgraph to have access to your custom qm, if you want to load the data, if you want to change the configuration or save backup locally.